packages/dd-trace/src/proxy.js

```diff
@@ -121,6 +108,27 @@ class Tracer extends NoopProxy {
     return this
   }
 
+  _enableOrDisableTracing (config) { // TODO: add test
```

Comment:
This should be tested. Enabling and disabling the tracer at runtime is a fairly big update and we should make sure it works properly.

Reply:
make sense, just added unit tests for it